ops/docker-compose-geth.yml

```diff
@@ -346,8 +347,8 @@ services:
       NO_NETWORK: 1
       L2_CHAINID: 31338 #unfortunately, elsewhere the L2_CHAINID is called CHAIN_ID
       << : *integration_pk
-  
-  # good idea, but never actually used? 
+
+  # good idea, but never actually used?
```

Comment:
we should eventually delete this commented out chunk 🗡️

Reply:
Do we do it this time ⚔️